apriltag_pose.h

```diff
@@ -17,9 +17,17 @@ typedef struct {
     double cy; // In pixels.
 } apriltag_detection_info_t;
 
+/**
+ * This struct holds the transformation from the camera optical frame to
+ * the April tag frame. The pose refers to the position of the tag within
+ * the camera frame.
+ *
+ * If you want to get the camera pose with respect to the April tag frame
+ * you have to apply the inverse of this transformation.
```

Comment:
I would say that this is redundant, i.e. this is common knowledge and applies to any transformation.